cmd/podman/manifest/rm.go

```diff
@@ -12,13 +12,13 @@ import (
 
 var (
 	rmCmd = &cobra.Command{
-		Use:               "rm LIST",
+		Use:               "rm [LIST...]",
```

Comment:
rm LIST [LIST ...]

Reply:
Currently, libpod only supports removing one image at a time.

Reply:
So `podman manifest rm` only can take one image as an arg? Then it should be rm LIST, but reading the code it looks like this command takes multiple args. Either case, this shouldn't be rm [LIST...] because that implies that rm doesn't need an arg, which it does.

Reply:
@rhatdan @ashley-cui Sorry, I confused remove with rm. Sorry, updating now.